From 7ba7440c49d32f911fb9e1c213307947a777085d Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Tue, 22 Aug 2023 12:43:02 -0400
Subject: [PATCH 35/35] compile_commands.json: remove stuff clang doesn't like

This is a "creature comfort" change to make it so gcc-specific options
don't make it into compile_commands.json.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile b/Makefile
index aaa927d..501c721 100644
--- a/Makefile
+++ b/Makefile
@@ -79,6 +79,9 @@ endif
 compile_commands.json : Makefile Make.rules Make.defaults 
 	make clean
 	bear -- make COMPILER=clang test all
+	sed -i \
+		-e 's/"-maccumulate-outgoing-args",//g' \
+		$@
 
 update :
 	git submodule update --init --recursive
-- 
2.39.0

